website/docs: rearranged brands docs (#14116)
goauthentik/authentik: website/docs/sys-mgmt/brands.md

---
title: Brands
slug: /brands
---

> As an authentik admin, you can control your instance's appearance and behaviour using brands. While a single authentik instance can only have a single brand, if you have different domains running for that instance, you can apply a separate brand to each domain.

Reviewer: ```suggestion
As an authentik admin, you can customize your instance's appearance and behavior using brands. While a single authentik instance supports only one brand per domain, you can configure separate brands for each domain your instance serves.
```

Author: took most of that, thanks!

> For an overview of branding and other customization options in authentik refer to [Customize your instance](../customize/index.md).

Reviewer: ```suggestion
For a broader overview of customization options in authentik, see [Customize your instance](../customize/index.md).
```

> To create or edit a brand, follow these steps:

Reviewer: ```suggestion
## Creating or editing a brand

To create or edit a brand, follow these steps:
```

Reviewer: Couldn't get multiline suggestions to work for some reason

Author: We want to avoid gerunds in titles/headers, but otherwise yes, good iea to add as a header, thanks @dominic-r !

> 1. Log in as an administrator, open the Admin interface, and navigate to **System** > **Brands**.

Reviewer: ```suggestion
1. Log in as an administrator, open the authentik Admin interface, and navigate to **System** > **Brands**.
```

Some places use this wording, some don't. We should make a final document with all these rules and add them to notion/here

Reviewer: actually, on the topic of "some places"... Line 30 uses the suggested wording ^^

> 2. Click **Create** to create a new brand, or click the edit icon beside an existing brand modify it.

Reviewer: ```suggestion
2. Click **Create** to add a new brand, or click the **Edit** icon next to an existing brand to modify it.
```

IMO

3. Define the configurations in the following settings:

### Branding settings

> The brand configuration controls the branding title (shown in website document title and several other places), the sidebar/header logo that appears in the upper left of the product interface, and the favicon on a browser tab.

Reviewer: I like having it like this, what do you think?

```suggestion
These settings define the visual identity of the brand, including:

- **Branding title**: Displayed in the browser tab (document title) and throughout the UI;
- **Logo**: Appears in the sidebar/header;
- **Favicon**: Shown on the browser tab.
```

:::info

> Starting with authentik 2024.6.2, the placeholder `%(theme)s` can be used in the logo configuration option, which will be replaced with the active theme.

Reviewer: Do we need to include this? 2024.6 is no longer supported

Author: I think I recall that the agreement is to keep such callouts for releases up to two years old. We only support the last two versions, but Jens wanted to keep these types of badges and mentions of specific versions for two years back.

:::

### External user settings

> You can configure authentik to redirect external users to a default application when they successfully authenticate (without being sent from a specific application). To do so, follow these steps:

Reviewer: ```suggestion
You can configure a brand to redirect unauthenticated users to a default application after they log in (if they weren't originally redirected from a specific application). To do this:
```

Author: On this case, we specifically mean external users, not unauthenicated ones. But the rest of the rewording I like and will keep, thanks! :-)
